WordPressShared/Core/Views/WPNoResultsView.m

```diff
@@ -7,8 +7,8 @@
 
 
 @interface WPNoResultsView ()
-@property (nonatomic, strong) UILabel   *titleLabel;
-@property (nonatomic, strong) UILabel   *messageLabel;
+@property (nonatomic, strong) UILabel       *titleLabel;
+@property (nonatomic, strong) UITextView    *messageTextView;
```

Comment:
OMG @elibud and @SergioEstevao LOOK!!! EDUARDO IS ONE OF MY KIND!!!!!!

Reply:
@etoledom ... We will need to have a talk... :)

Reply:
😱😆